packages/calcite-components/src/components/tab-nav/tab-nav.scss

```diff
@@ -41,6 +42,20 @@
     ease-out;
 }
 
+calcite-action {
+  @apply absolute;
+  inset-block-start: 50%;
+  transform: translateY(-50%);
+}
+
+.chevron-right {
```

Comment:
question: for these `arrowRight` and `chevron-right` should we be using terms `chevron-end` and `arrowEnd` instead?

Reply:
Hm... I don't think so. I think start/end vocabulary is meant to indicate to a user where they are slotting their actions or icons. In this case, action is built in and conditionally rendered, and that naming is probably irrelevant.

Reply:
Do these flip in RTL direction? If so then "right" isn't right anymore.

The start/end vocabulary is used to describe placement independent of direction (right/left) for page directions that are flipped.

Reply:
Hm.. you're right. It would only flip if we have tab order flipping. Let me check.

Reply:
It looks like it does swap the direction, so that was a good one to bring up. This makes sense, because tabs might be numbered and ordered by the user.

Reply:
This is a good point that we should pass along to the icon team. Are there hesitations with asking them to rename all icons with reference to right/left in favor of start/end?

Reply:
I don't think the icon names are an issue, just the CSS class names and object properties.

Reply:
Oh missed that - thanks! 😁